### DISCUSSION TITLE
Video start full screen on Iphone devices

### DISCUSSION THREAD

**Discussion by Scippy** [videojs/video.js#8624] [Q&A] [answered]
Hi, my video on Iphone devices start fullscreen and I'm no able to play it inside div element.
I've try with preferFullWindow option and with vjs-nofull class but the video when played go full screen!
How I can solve it?

**Comment by Scippy** [comment, marked as answer]
I've solved with playsinline="" option!

**Reply by amtins** [reply]
The direct link to the documentation, in case anyone else is looking for it and comes across this discussion https://videojs.com/guides/options/#playsinline
